tracks/ruby/exercises/high-scores/mentoring.md

```diff
@@ -0,0 +1,71 @@
+New concepts: Array query methods (+ chaining methods), instantiating a class, getter method or `attr_reader`.
+
+### Minimal solution for approval
+
+```ruby
+
+# version 2+
```

Comment:
I would suggest removing `# version 2+`, the version should be bound either by the `Gemfile`, a `.ruby-version` or whatever -- I don'think you want to deal with n versions of solutions in the end in the mentor notes, which is what this implies to me.

Reply:
Actually, I meant the version of the exercise. But as it's confusing, obviously, and there's a changelog, I'll remove it.